--- src/xercesc/parsers/SAXParser.hpp.orig	2004-04-05 16:20:21.000000000 -0400
+++ src/xercesc/parsers/SAXParser.hpp	2004-04-05 16:28:18.000000000 -0400
@@ -289,6 +289,14 @@
     , public DocTypeHandler    
 {
 public :
+    /**
+      * This method returns a reference to the underlying scanner object.
+      * It allows read only access to data maintained in the scanner.
+      *
+      * @return A const reference to the underlying scanner object.
+      */
+    const XMLScanner& getScanner() const;
+
     // -----------------------------------------------------------------------
     //  Class types
     // -----------------------------------------------------------------------
@@ -2059,7 +2067,8 @@
       *
       * @return A const reference to the underlying scanner object.
       */
-    const XMLScanner& getScanner() const;
+//    const XMLScanner& getScanner() const;
+// getScanner() moved to public section
 
     /** Get the Grammar resolver
       *
